README.md

```diff
@@ -109,28 +109,28 @@ JSON modules' semantics are those of a single default export which is the entire
+Hosts would all be required to give a common interpretation to `"json"`, defined in the JavaScript specification, that `json` is the parsed JSON document (no named exports). Further conditions and module types beyond `json` modules could be added in future TC39 proposals as well as by hosts. HTML and CSS modules are also under consideration, and these may use similar explicit `type` syntax when imported.
 
-JavaScript implementations are encouraged to reject attributes and type values which are not implemented in their environment (rather than ignoring them). This is to allow for maximal flexibility in the design space in the future--in particular, it enables new import attributes to be defined which change the interpretation of a module, without breaking backwards-compatibility.
+JavaScript implementations are encouraged to reject conditions and type values which are not implemented in their environment (rather than ignoring them). This is to allow for maximal flexibility in the design space in the future--in particular, it enables new import conditions to be defined which change the interpretation of a module, without breaking backwards-compatibility.
 
 Note that all environments are required to support JSON modules with this explicit syntax, but *may* support modules without it. For example, on the Web, JSON modules would only be supported with the explicit type, but Node.js *may* decide to also support JSON modules without this declaration. However, all environments *must* support the explicitly `type`-declared JSON modules.
 
-### Restriction to "check attributes"
+### Restriction to conditions attributes
 
-The restriction of attributes to not affect the contents of the module or be part of the cache key is sometimes referred to as permitting only "check attributes" and not "evaluator attributes", where the latter would change the contents of the module. Future versions of this specification may relax this restriction, and it's understood that some hosts may be tempted to willfully violate this restriction, but the module attributes champion group advises caution with such a move. There are three possible ways to handle multiple imports of the same module with different attributes, if the attributes cause a change in the interpretation of the module:
+Conditions do not affect the contents of the module or be part of the cache key, and it's understood that some hosts may be tempted to willfully violate this restriction, but the module conditions champion group advises caution with such a move. Future follow-up proposals may relax this restriction with "evaluator attributes" that would change the contents of the module. There are three possible ways to handle multiple imports of the same module with different conditions, if the conditions cause a change in the interpretation of the module:
```

Comment:
Let's reframe this section to be about the challenges the `with` follow-on proposal would face. It doesn't make sense to call these conditions.

Reply:
shouldn't move that section to the follow-up proposal? since we explictly don't talk about evaluator attributes anymore here

Reply:
Yes, that'd be ideal; we can cross-reference that doc from here. Initially, we can just edit the text that's in this document.